teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Teacher < Person
+  def initialize(age:, specialization:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] Also here kindly remove the `:` after age and specialization

```suggestion
  def initialize(age, specialization, name: 'Unknown', parent_permission: true)
```